graphistry/feature_utils.py

```diff
@@ -2608,3 +2648,76 @@ def _featurize_or_get_edges_dataframe_if_X_is_None(
             reuse_if_existing=True,
             memoize=memoize,
         )
+        
+    def get_matrix(self, column, kind='nodes'):
```

Comment:
let's give it a name that makes sense to users scrolling through, like `.features_by_cols(cols=[...])`

Reply:
changed this to `get_features_by_cols(self, columns: Union[List, str], kind: str = 'nodes')`